bpfd/src/server/bpf.rs

```diff
@@ -333,6 +341,11 @@ impl<'a> BpfManager<'a> {
         }
         Ok(())
     }
+
+    fn remove_dispatcher(&mut self, iface: String) -> Result<(), BpfdError> {
```

Comment:
Since this is one line we may as well just inline it to save adding another function.

Reply:
Using `#[inline(always)]` or meaning just get rid of the function and move the `self.dispatchers.remove(&iface);` call above?

Reply:
Done using `#[inline(always)]`.